src/node_http2.h

```diff
@@ -7,6 +7,7 @@
 #include <cstdint>
 #include "nghttp2/nghttp2.h"
 
+#include "aliased_struct-inl.h"
```

Comment:
It should be enough to include `aliased_struct.h` here and include the `-inl.h` header from the `.cc` file only, right? Otherwise there’s something else here that needs fixing…

Reply:
Oy doh! Yeah actually meant to do that. Thanks for catching